### ISSUE TITLE
mdtraj.Trajectory.__init__() got an unexpected keyword argument 'unitcell_vectors'

### ISSUE THREAD

**Issue by xiki-tempula** [mdtraj/mdtraj#1772]
The doc in https://mdtraj.org/1.9.4/api/generated/mdtraj.Trajectory.html mentioned that 
`and information about the unitcell if appropriate (unitcell_vectors, unitcell_length, unitcell_angles).`

However, `unitcell_vectors` is not a keyword argument for `mdtraj.Trajectory.__init__`.

**Comment by mattwthompson** [comment]
Good catch - I think this should be a change in the docs since the source code doesn't use `unitcell_vectors` in the constructor. I checked the latest version (I think 1.9.7) to be sure.
